src/v2/Apps/Order/Components/AddressModal.tsx

```diff
@@ -102,10 +104,21 @@ export const AddressModal: React.FC<Props> = ({
               onError && onError(message)
             }
 
-            const handleSuccess = address => {
+            const handleSuccess = savedAddress => {
+              if (isDefault) {
```

Comment:
Blocking if my understanding is correct: Do we not have access to the value through `values.isDefault`? It looks like we should ([here's the underlying type](https://github.com/artsy/force/blob/1db17a9b7ee6ff5c44401e0b46160d04cde68ab0/src/v2/__generated__/Shipping_me.graphql.ts#L20)). If we do, we should use that instead of creating another state variable to manage this value.

Reply:
fixed